examples/amp-first/.gitignore

```diff
@@ -0,0 +1,26 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I think the .gitignore can be removed, similar to other examples.

Reply:
done